Fix tiltrotor model and add to SITL testing

Jaeyoung-Lim opened the pull request:
This PR fixes the tiltrotor model that was failing SITL testing before, and adds the tests to mavsdk testing

Jaeyoung-Lim committed:
Fix tiltrotor model

Jaeyoung-Lim committed:
Add tiltrotor to SITL testing

TSC21 commented on models/tiltrotor/tiltrotor.sdf:
```diff
@@ -1035,9 +1028,15 @@
-    <plugin name='groundtruth_plugin' filename='libgazebo_groundtruth_plugin.so'>
-      <robotNamespace/>
-    </plugin>
+    <include>
```
I would actually prefer this where it was since here it's like in the middle of the plugins.

Jaeyoung-Lim replied:
Done

Jaeyoung-Lim committed:
Move where the gps joint is being defined

Move where the gps joint is being defined inside the sdf file, based on the review comments

Jaeyoung-Lim committed:
Switch tiltrotor to jinja templates
